lib/src/main/java/com/androidessence/materialdesignspecs/ColorDialog.java

```diff
@@ -64,61 +79,32 @@ public void setOnColorSelectedListener(OnColorSelectedListener listener) {
         this.listener = listener;
     }
 
-    public interface OnColorSelectedListener {
-        void onColorSelected(Integer color);
+    public void setLayoutManager(RecyclerView.LayoutManager layoutManager) {
+        this.layoutManager = layoutManager;
```

Comment:
Do we need to reattach the layoutmanager to the recyclerview here? Or will it handle itself because it's by reference?